lib/solidus_dev_support/templates/extension/gem_release.yml.tt

```diff
@@ -2,4 +2,5 @@ bump:
   recurse: false
   file: 'lib/<%=file_name%>/version.rb'
   message: Bump <%=class_name%> to %{version}
-  branch: true
+  branch: false
```

Comment:
I think `false` is the default value, do you think it's better to keep it explicit here?

Reply:
@kennyadsl fixed!